mysql-exporter/mysql-exporter.libsonnet

```diff
@@ -10,13 +10,20 @@ local deployment = k.apps.v1.deployment;
     mysql_password: error 'must specify mysql password',
     deployment_name: error 'must specify deployment name',
     namespace: error 'must specify namespace',
+    mysql_fqdn: '',
```

Comment:
I suggest you make this a hidden variable like `image` unless it's a required param.